modin/data_management/functions/groupby_function.py

```diff
@@ -141,13 +141,17 @@ def caller(
         cls,
         query_compiler,
         by,
-        axis,
-        groupby_args,
-        map_args,
         map_func,
+        axis=0,
```

Comment:
I prefer to revert all of these changes and go through the `groupby` in the API layer.